exercises/diffie-hellman/Hints.md

```diff
@@ -0,0 +1,3 @@
+## Hints
+This exercise requires you to perform calculations with the large numbers. It might be a good idea to use [BigIntegers]
```

Comment:
"It might be a good idea to use [BigIntegers]" should be "To correctly represent large numbers, the [BigInteger](https://msdn.microsoft.com/en-us/library/system.numerics.biginteger(v=vs.110).aspx) struct is used."